src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Add-AzsScaleUnitNode.ps1

```diff
@@ -26,29 +26,33 @@ Licensed under the MIT License. See License.txt in the project root for license
 function Add-AzsScaleUnitNode
```

Comment:
Do we have test coverage for this

Reply:
lol no.  I don't have a 20K server to test it.  We will test internally in CI and have OEM partners test.